sbomify/apps/teams/templates/teams/dashboard.html.j2

```diff
@@ -1,6 +1,11 @@
-{% extends "core/dashboard_base.html.j2" %}
+{% extends "core/dashboard_base.htmx.j2" %}
```

Comment:
We need to reconfigure the Django template linting (both in git hooks and CI) to include these new files.

Reply:
Resolved